bb8/src/api.rs

```diff
@@ -1,15 +1,21 @@
+use crate::inner::PoolInner;
+use crate::internals::Conn;
+pub use crate::internals::State;
+use async_trait::async_trait;
 use std::borrow::Cow;
 use std::error;
 use std::fmt;
+use std::future::Future;
 use std::marker::PhantomData;
 use std::ops::{Deref, DerefMut};
+use std::pin::Pin;
 use std::time::Duration;
 
-use async_trait::async_trait;
-
-use crate::inner::PoolInner;
-use crate::internals::Conn;
-pub use crate::internals::State;
+/// An executor of futures.
```

Comment:
I'd like this definition to go at the bottom of the module, with the `struct TokioExecutor` definition directly above it.